kaldb/src/test/java/com/slack/kaldb/chunk/ChunkCleanerServiceTest.java

```diff
@@ -53,6 +54,8 @@ public void setUp() throws Exception {
     metricsRegistry = new SimpleMeterRegistry();
     chunkManagerUtil =
         new ChunkManagerUtil<>(S3_MOCK_RULE, metricsRegistry, 10 * 1024 * 1024 * 1024L, 10L);
+    chunkManagerUtil.chunkManager.startAsync();
```

Comment:
We changed the semantics of `ChunkManagerUtil` ( a test helper ) to not start chunk manager

Because of this we need to start it manually everywhere

The reason for the change was in `KaldbDistributedQueryServiceTest` we need to put chunk manager in a service manager ( which will not allow a running service to be put in the SM )